components/eam/src/physics/rrtmg/radiation.F90

```diff
@@ -708,7 +714,8 @@ subroutine radiation_init(phys_state)
           call addfld('FLUT'//diag(icall), horiz_only,    'A',    'W/m2', 'Upwelling longwave flux at top of model', &
```

Comment:
FLUT is more commonly used than FLUTC, so it is useful to include standard name, 'toa_outgoing_longwave_flux"

Similarly quite a lot attention are paid to SWCF and LWCF for assess model performance involving cloud processes.  Including their standard_names should also be useful
SWCF: toa_shortwave_cloud_radiative_effect
LWCF: toa_longwave_cloud_radiative_effect

Reply:
As I said in the description, this only adds standard names for variables we published already AND require no modifications.  Adding more standard_names can be part of a future PR or you can add them to this one today if you want.

Reply:
Sounds good, Rob. I was about to ask if we want to add for more variables in this PR. There will be more than just the few I mentioned, I think a future PR is better, Thanks for the explanation.

Reply:
@rljacob : there probably won't be time for another PR.

Reply:
Right.  I meant another PR sometime in the future.   It will take some time to figure out other standard_names.

Reply:
@golaz , @rljacob : I am updating this PR with additional standard names.

Reply:
Only do the ones that are really obvious and it has to be done before next testing starts tonight.